app/notifications/receive_notifications.py

```diff
@@ -60,10 +60,10 @@ def receive_firetext_sms():
     auth = request.authorization
     if not auth:
         current_app.logger.warning("Inbound sms no auth header")
-        # abort(401)
+        abort(401)
```

Comment:
When both providers have authorization, will we migrate this to be handled by the auth similar to the other endpoint validations?

Reply:
Yeah, we can move it `before_request` if they end up with a similar authentication method.